pkg/cli/repo_cmd.go

```diff
@@ -0,0 +1,103 @@
+//go:build extended
+// +build extended
+
+package cli
+
+import (
+	"os"
+	"path"
+
+	"github.com/briandowns/spinner"
+	"github.com/spf13/cobra"
+	zotErrors "zotregistry.io/zot/errors"
+)
+
+func NewRepoCommand(searchService SearchService) *cobra.Command {
+	var servURL, user, outputFormat string
```

Comment:
need to tell cobra to populate these variables

repoCmd.Flags().StringVar(&servURL, "url", "", "Specify zot server URL if config-name is not mentioned")
repoCmd.Flags().StringVarP(&user, "user", "u", "", `User Credentials of zot server in "username:password" format`)

Reply:
done at lines 95-96 in repo_cmd.go